test/plugins.js

```diff
@@ -231,15 +247,33 @@ describe('Plugins', function () {
   describe('other options', function () {
     it('should modify options in plugin runtime', function () {
       var html = '<div class="cls"><br><rect></div>'
-      var ref = '<div class="cls"><br /><rect /></div>'
+      var ref = '<div class="cls"><br>< /><rect></rect></ /></div>'
```

Comment:
yep o_0 indeed, `messages` should have absolute no effect on this test, I made a stupid mistake somewhere, but still clueless where&why, need to investigate in this again, any idea why it happens?